full_snapshot.proto

```diff
@@ -458,6 +477,11 @@ message RelationStatistic {
   int64 tidx_blks_hit = 25;       // Number of buffer hits in this table's TOAST table indexes (if any)
   int64 toast_size_bytes = 26;    // TOAST table and TOAST index size (included in size_bytes as well)
   NullTimestamp analyzed_at = 27;
+
+  uint32 min_xid = 30; // Oldest transaction ID for this relation
+  uint32 min_mxid = 31; // Oldest multixact ID for this relation
```

Comment:
I think we should add a "frozen_" prefix here, assuming that's what we are referring to.

Additionally, I wonder if we should use 64-bit transaction IDs here? (we'd need some logic that looks at the current TXID counter and sets the correct epoch based on the counter value)

Reply:
Talked in Slack, we decided to drop `min_xid` and `min_mxid` from here in favor of other message we try to introduce shortly.